backend/packages/Upgrade/src/api/services/FeatureFlagService.ts

```
@@ -1289,4 +1289,43 @@ export class FeatureFlagService {
       compatibilityType: compatibilityType,
     };
   }
+
+  public async exportAllLists(
+    id: string,
+    listType: FEATURE_FLAG_PARTICIPANT_LIST_KEY,
+    logger: UpgradeLogger
+  ): Promise<ImportFeatureFlagListValidator[] | null> {
+    const featureFlag = await this.findOne(id, logger);
+    let listsArray: ImportFeatureFlagListValidator[] = [];
+    if (featureFlag) {
```

We should throw an error if the feature flag doen't exist.